### ISSUE TITLE
New Core Feature: Edit posts

### ISSUE THREAD

**Issue by y2kbadbug** [funkybob/django-dequorum#38]
Ability to edit a post once already made.  Could be combined with the create post page and just have it pull information from the DB.

**Comment by funkybob** [comment]
To support this we'd need an edit history... otherwise people can [and will] play games with trolling then removing the evidence, making the aggrieved respondents look like the aggressors.

**Comment by y2kbadbug** [comment]
Definitely.  Admins or moderators could possibly have access to the prior posts.  Its also good to restrict a post from being edited until it is 5 minutes old.
